### DISCUSSION TITLE
Elements structure: parent or child elements of .applet-box

### DISCUSSION THREAD

**Discussion by danimesq** [linuxmint/discussions#12] [Development]
Are there **empty** spare child or parent elements of .applet-box that could be customized to have margin and border-radius?

That could enable advanced theming, such as the examples bellow:

`.applet-box{margin-top:2px;margin-bottom:2px;border-radius:5px}`

![Screenshot from 2023-05-04 15-15-46](https://user-images.githubusercontent.com/6765277/236294038-4aead266-334d-43ea-bd04-2c74fdbed76f.png)

![Screenshot from 2023-05-04 15-15-42](https://user-images.githubusercontent.com/6765277/236294033-398441f6-e5db-4d96-ac5f-9466e73701b8.png)

![Screenshot from 2023-05-04 15-15-58](https://user-images.githubusercontent.com/6765277/236294041-ed84485e-ea01-46b2-8f92-dabd7cfc432f.png)

Currently in my case, Cinnamon's elements structure don't seem fully support this kind of CSS, as out-of-margin and out-of-border parts won't be hoverable (because I saw no parent element to hold the .applet-box nor child element to emulate it).

**Comment by danimesq** [comment]
Could `::before` and/or `::after` styling be a workaround?

**Reply by clefebvre** [reply]
I'm not sure myself. @mtwebster @JosephMcc any idea?
